README.md

```diff
@@ -1,20 +1,23 @@
-# Dash Cluster Ansible
+# Dash Networks Deployment Tool
```

Comment:
probably "Network" makes more sense

Reply:
I thought if we are deploying multiple networks we should keep it plural. No?
If I'm wrong we should rename repo too.

Reply:
It's per-network. Makes more sense in English to call it a "network deployment tool" -- it is understood that the tool can be used over & over. But you can call it what you want.

Reply:
Ok. I see. Then I will fix it everywhere.